]), axis=0)[0]

```diff
@@ -0,0 +1,236 @@
+
+                   SSUUMMMMAARRYY OOFF LLEESSSS CCOOMMMMAANNDDSS
```

Comment:
@johnsam7 if you're trying to check if the reviewers are bots, you will be disappointed :P

Reply:
hahaha wth?!